R/tm_a_regression.R

```diff
@@ -1,20 +1,27 @@
-#' Scatterplot and Regression Model
-#' @md
+#' Scatterplot and regression model
+#'
+#' Module for visualizing regression analysis, including scatterplots and
+#' various regression diagnostics plots.
+#' It allows users to explore the relationship between a set of regressors and a response variable,
+#' visualize residuals, and identify outliers.
+#'
+#' @note For more examples, please see the vignette "Using regression plots" via
+#' `vignette("using-regression-plots", package = "teal.modules.general")`.
 #'
 #' @inheritParams teal::module
 #' @inheritParams shared_params
 #' @param regressor (`data_extract_spec` or `list` of multiple `data_extract_spec`)
-#'  Regressor variables from an incoming dataset with filtering and selecting.
+#' Regressor variables from an incoming dataset with filtering and selecting.
 #' @param response (`data_extract_spec` or `list` of multiple `data_extract_spec`)
-#'  Response variables from an incoming dataset with filtering and selecting.
-#' @param alpha optional, (`numeric`) If scalar then the plot points will have a fixed opacity. If a
-#'   slider should be presented to adjust the plot point opacity dynamically then it can be a vector of
-#'   length three with `c(value, min, max)`.
-#' @param size optional, (`numeric`) If scalar then the plot point sizes will have a fixed size
-#'   If a slider should be presented to adjust the plot point sizes dynamically then it can be a
-#'   vector of length three with `c(value, min, max)`.
-#' @param default_outlier_label optional, (`character`) The default column selected to label outliers.
-#' @param default_plot_type optional, (`numeric`) Defaults to Response vs Regressor.
+#' Response variables from an incoming dataset with filtering and selecting.
+#' @param alpha (`numeric`, optional) Specifies point opacity.
+#' - If vector of `length == 1` then the plot points will have a fixed opacity.
+#' - while vector of `value`, `min`, and `max` allows dynamic adjustment.
+#' @param size (`numeric`, optional) Specifies point size.
```

Comment:
Let us follow the same convention for the arg document what we use in the PR #669

Reply:
Can we have a similar wording as we did in that PR?

```r
#' - When the length of `bins` is one: The histogram bins will have a fixed size based on the `bins` provided.
#' - When the length of `bins` is three: The histogram bins are dynamically adjusted based on vector of `value`, `min`, and `max`.
```

Reply:
@kartikeyakirar After this change we can merge this one too.

Reply:
[9186292](https://github.com/insightsengineering/teal.modules.general/pull/655/commits/91862926e01ae4dae86c215fb588ce0354705578)